### PR TITLE
{geo}[intel/2016a] rgeos 0.3-17 w/ R 3.2.3 REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2151.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3f44763531e8b1ce573b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0ed06f73821359d737c1 for a full test report.

File: easybuild/easyconfigs/s/SWIG/SWIG-3.0.8-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = 'SWIG'
+version = '3.0.8'
+versionsuffix = '-Python-2.7.11'
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SWIG/SWIG-3.0.8-intel-2016a-Python-2.7.11.eb]: `pyver`

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**